docs/src/app/(new)/page.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import { Metadata } from 'next/types';
+import { readFile } from 'node:fs/promises';
+import './page.css';
+
+const description =
+  'Unstyled React components for building accessible user interfaces.';
+
+export default async function Page() {
+  const logo = await readFile('public/static/logo.svg', 'utf-8');
+  return (
+    <div className="relative flex grow flex-col justify-between gap-10 lg:justify-center [@media(min-height:700px)]:justify-center">
+      <div className="flex px-7 pt-7 md:px-16 md:pt-14 lg:items-center lg:pb-[min(10rem,max(3.5rem,10vh))]">
+        <div className="max-w-[26rem] 2xl:max-w-[40rem]">
+          <h1 className="mb-2 text-balance text-2xl font-medium 2xl:mb-3 2xl:text-3xl">
+            {description}
+          </h1>
+          <p className="text-color-gray mb-4 text-pretty 2xl:text-lg">
+            From the creators of Radix, Floating UI, and MUI.
```

Comment:
```suggestion
            From the creators of Radix, Floating UI, and Material UI.
```

Reply:
@vladmoroz It will be MUI next year, but yeah it should be "Material UI" for now.

Reply:
> @vladmoroz It will be MUI next year, but yeah it should be "Material UI" for now.

Looks like I also don't understand the difference between MUI and Material UI 🫠
How come it will change next year? Material UI library will be renamed to MUI?

Reply:
Current state:
MUI - the company
Material UI - the library

Who knows what the future will bring ;)